packages/tables/src/Filters/SelectFilter.php

```diff
@@ -49,6 +49,7 @@ protected function setUp(): void
                     $relationshipQuery = $filter->getRelationshipQuery();
 
                     $labels = $relationshipQuery
+                        ->distinct()
```

Comment:
This is going to reintroduce a problem we fixed before, we probably need to check the relationship type first

Reply:
Thanks for the feedback, I added a check before applying the distinct().